bouncer/blacklist/tests.py

```diff
@@ -33,6 +36,11 @@ def setUpClass(cls):
         )
         upper_case_email_entry.save()
 
+        user = User(username="example_user")
+        cls.authenticated_client = APIClient()
+        cls.authenticated_client.force_authenticate(user=user)
```

Comment:
Please use `force_authenticate(token=XXX)` instead, as this is the authentication method we'd like to test.